test/e2e/image.test.js

```diff
@@ -0,0 +1,11 @@
+
+const { assert, driver } = require('vl-ui-core').Test;
+const VlImagePage = require('./pages/vl-image.page');
+
+describe('vl-image', async () => {
+    const vlImagePage = new VlImagePage(driver);
+
+    before(() => {
+        return vlImagePage.load();
+    });
+});
```

Comment:
moet ge hier niet checken dat den image er is dan?

Reply:
de load opent enkel de pagina. Het al dan niet checken op elementen moet in de functies gedaan worden die het element effectief nodig hebben.